plugins/kubernetes/src/components/JobsAccordions/JobsDrawer.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { V1Job } from '@kubernetes/client-node';
+import { KubernetesDrawer } from '../KubernetesDrawer/KubernetesDrawer';
+import { Typography, Grid } from '@material-ui/core';
+
+export const JobDrawer = ({
+  job,
+  expanded,
+}: {
+  job: V1Job;
+  expanded?: boolean;
+}) => {
+  return (
+    <KubernetesDrawer
+      object={job}
+      expanded={expanded}
+      kind="Job"
+      renderObject={(jobObj: V1Job) => {
+        return {
+          parallelism: jobObj.spec?.parallelism ?? '???',
+          completions: jobObj.spec?.completions ?? '???',
+          backOffLimit: jobObj.spec?.backoffLimit ?? '???',
```

Comment:
perhaps unfortunate to have another casing for backOff (capital "O") than the official source data has.

Reply:
good catch!